radis/tools/slit.py

```diff
@@ -1630,7 +1630,7 @@ def triangular_slit(
     return w, I
 
 
-# trapezoidal instrumental broadening function of base base nm and top top nm
+# trapezoidal instrumental broadening function of base (nm) and top (nm)
```

Comment:
In general I like my duplicate detection... This case is interesting. I think it's trying to describe the identify of things, their variables, and their types. I don't think writing it that way is helpful and so I'm proposing an alternative. I'm not attached to it. My tool can be tuned to ignore this content...